app/models/comment.rb

```diff
@@ -1,4 +1,16 @@
 class Comment < ApplicationRecord
   belongs_to :user
   belongs_to :post
+
+  after_create :increment_post_comment_counter
+
+  private
+
+  def increment_post_comment_counter
+    post.increment_comment_counter
+  end
+
+  def increment_comment_counter
+    update(comment_counter: comment_counter + 1)
+  end
```

Comment:
- Please ensure that you are properly updating the specified attribute to avoid errors in your code. Please fix this 🙏

Reply:
done